docs/.vitepress/config.js

```diff
@@ -2,13 +2,12 @@
 
 /** @type {UserConfig} */
 const config = {
-  // base: '/v2/',
   title: 'Vue Test Utils for Vue 3',
   description: 'The documentation for the official Vue Test Utils',
   locales: {
     '/': {
       lang: 'en-US',
-      title: 'Vue Test Utils (2.0.0-beta.14)'
+      title: 'Vue Test Utils (2.0.0-rc.0)'
```

Comment:
We could actually import `package.json` here to make sure this is always up to date

Reply:
sure, done!